src/main/java/mate/academy/bookstore/dto/user/UserLoginRequestDto.java

```diff
@@ -0,0 +1,20 @@
+package mate.academy.bookstore.dto.user;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.Size;
+import lombok.Data;
+
+@Data
+public class UserLoginRequestDto {
+    @NotEmpty
```

Comment:
@NotEmpty and @NotBlank are mostly useless here as you anyway validate the length of string in Size annotation. So you can replace @NotEmpty and @NotBlank  by @NotNull